### DISCUSSION TITLE
Comando Console.Clear(); dá erro de build.

### DISCUSSION THREAD

**Discussion by JonathanBarross** [balta-io/forum#532] [Tira Dúvidas] [answered]
Olá, estou trabalhando na atividade prática da calculadora em C#, que está disponível ainda no módulo de fundamentos. Sempre que tento usar o comando "Console.Clear();'", o aplicativo retorna um erro e acaba travando o funcionamento do programa que criei. Assim que removo o comando, ele volta a funcionar normalmente. Gostaria de saber como eu poderia resolver esse erro.
![image](https://github.com/balta-io/forum/assets/126283157/d311dff3-efd0-4dc5-8638-9afd5f13e4d6)

**Comment by henriqueweege** [comment, marked as answer]
Oi, @ThePatienceOne ! Tudo bem?
Você teria como subir teu projeto no Git para eu dar uma olhada? 
Valeeeu!
